src/itype.h

```diff
@@ -67,7 +67,7 @@ struct islot_comestible
     std::string tool = "null";
 
     /** Defaults # of charges (drugs, loaf of bread? etc) */
-    long def_charges = 0;
+    long def_charges = 1;
```

Comment:
These shouldn't exist and a quick check with `jq` confirms there are no items dependent upon a default value of `charges 0`